charts/brig/templates/configmap.yaml

```
@@ -12,6 +12,7 @@ data:
   brig.yaml: |
     logNetStrings: True # log using netstrings encoding:
                         # http://cr.yp.to/proto/netstrings.txt
+    logFormat: JSON
```

Should the `logNetStrings` lines above be removed, then?

---

We can't yet.  Also see https://github.com/wireapp/wire-server-deploy/issues/143

The current way we use configmaps we must make sure that the old and the new version of the deployment can both parse the config or bad things will happen on rollback or on rollout failure. 

We can remove it once we have rolled out all the deployments successfully.

---

```suggestion
    logFormat: {{ .logFormat }}
```

So that we can override it with default values